res/database/Default/user/therob.xml

```diff
@@ -2609,87 +2943,101 @@
 			<title>
 				<de>TVTs: %OBJECT% im Hochhaus</de>
 				<en>TVTs: %OBJECT% In A Skyscraper</en>
+				<pl>TVT: %OBJECT% w wieżowcu</pl>
 			</title>
 			<description>
 				<de>Serie über %OBJECT% im TV Tower Hochhaus</de>
 				<en>Series about %OBJECT% in The TV Tower Skyscraper</en>
+				<pl>Serial. %OBJECT% w wieżowcu telewizyjnym TV Tower</pl>
 			</description>
 
 			<children>
 				<scripttemplate index="0" product="2" licence_type="2" guid="TheRob-script-ser-dram-TVTintern-Ep1">
 					<title>
 						<de>TVTs Pilot</de>
 						<en>TVTs Pilot</en>
+						<pl>TVTs Pilot</pl>
```

Comment:
Just wondering: here TVTs, but in the series Title you left out the "s" explicitly.

Reply:
You're right. An oversight. Corrected.